README.md

```diff
@@ -4,6 +4,7 @@
 [![CLA Assistant](https://github.com/OpenZeppelin/openzeppelin-relayer/actions/workflows/cla.yml/badge.svg)](https://github.com/OpenZeppelin/openzeppelin-relayer/actions/workflows/cla.yml)
 [![CI](https://github.com/OpenZeppelin/openzeppelin-relayer/actions/workflows/ci.yaml/badge.svg)](https://github.com/OpenZeppelin/openzeppelin-relayer/actions/workflows/ci.yaml)
 [![Release Workflow](https://github.com/OpenZeppelin/openzeppelin-relayer/actions/workflows/release-please.yml/badge.svg)](https://github.com/OpenZeppelin/openzeppelin-relayer/actions/workflows/release-please.yml)
+[![Scorecard](https://github.com/OpenZeppelin/openzeppelin-relayer/actions/workflows/scorecard.yml/badge.svg)](https://github.com/OpenZeppelin/openzeppelin-relayer/actions/workflows/scorecard.yml)
```

Comment:
This is fine , but we should add openssf scorecard badge just like https://github.com/OpenZeppelin/defender-sdk/blob/main/README.md?plain=1#L6C23-L6C109